fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_InitPage.js

```diff
@@ -373,7 +373,9 @@ class CDUInitPage {
             if (value === "") {
                 let zfw = undefined;
                 let zfwCg = undefined;
-                if (!!SimVar.GetSimVarValue("L:A32NX_BOARDING_STARTED_BY_USR", "bool")) {
+                let a32nxBoarding = !!SimVar.GetSimVarValue("L:A32NX_BOARDING_STARTED_BY_USR", "bool");
```

Comment:
Unsure if !! is actually necessary  here but it's pulled from the existing logic so maybe there was some purpose to it

Reply:
I am pretty sure it works without, since we use it all over in the code. No clue what the motivation for it was for @tracernz